docs/readmes/basics/prerequisites.md

```diff
@@ -140,14 +145,15 @@ Development can occur from multiple OS's, where **macOS** and **Ubuntu** are **e
    2. Install the following dependencies
 
       ```bash
-      pip3 install ansible fabric3 jsonpickle requests PyYAML
+      pip3 install ansible 'fabric<2.0' jsonpickle requests PyYAML
```

Comment:
Why do you have to install fabric < 2.0?

Reply:
Thanks, yes upon revisiting this it appears I misunderstood something about the different fabric versions that exist. I reverted this change.